lib/logger/test/logger_test.exs

```diff
@@ -760,7 +784,7 @@ defmodule LoggerTest do
 
     test "maps Erlang levels" do
       :logger.set_primary_config(:level, :notice)
-      assert capture_log(fn -> Logger.info("hello") end) =~ "hello"
+      assert capture_log(fn -> Logger.warning("hello") end) =~ "hello"
```

Comment:
I'm not sure what the behaviour was being tested here. This started failing when I made `capture_log` not change log level